Sources/SLaM/CLI.swift

```diff
@@ -22,11 +22,12 @@ public struct CLI {
         
         switch arguments[1] {
         case "new":
+            if !checkPrerequisites() {
+                exit(-1)
+            }
             try createNewLambdaProject()
-            
-        case "setup-image":
```

Comment:
when we will have a community using this tool 😉, this will be mandatory. But right now it is just you and me :-) I don't want to start with an history. I would suggest to to start clean and with current language and Ones support.